TRIAGING.md

```diff
@@ -0,0 +1,89 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
+The maintainers of the k-NN/neural-search Repo's seek to promote an inclusive and engaged community of contributors. In 
```

Comment:
Can we separate meetings between k-NN and neural-search? I think more focused meeting for each repo could be more efficient. Two groups has different maintainers and different interests I believe.

Reply:
Mentioning of triaging issue `neural-search` repo in k-NN repo sounds a bit weird. Not sure what others think. 
How can we handle if there are many issues in knn and we use most of the time on knn issue only? Should we divide it like 8:00AM to 8:30AM to knn and 8:30AM to 9:00AM to neural? Some people might only have interest in one of repo but not both and to save their time clear separation might be good.

Reply:
Makes sense. How about we focus every alternate week for each plugin? Not sure if 30 min is enough

Reply:
If we want to separate the meetings, we can decide the frequency and length of the meetings on a per-repo basis. I agree that 30 minutes might not be enough.

Reply:
We had a discussion with the team and everyone said to have 1 bi-weekly meeting for the 2 repos. I think we should start out with this plan and then adjust as we see fit in the future.

Reply:
Fair enough.